src/main/resources/changesets/changelog-consent-67.0.xml

```diff
@@ -0,0 +1,59 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="67.0" author="raejohanek">
+        <createTable tableName="library_card">
+            <column name="id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="user_id" type="bigint">
+            </column>
```

Comment:
minor nit: here and throughout, self-close tags when they don't have any inner content:
```suggestion
            <column name="user_id" type="bigint"/>
```